hedyorg/hedy
# Ru add students in class overview

**RU / New Media Lab issue**

Fixes To Do mentioned in https://github.com/hedyorg/hedy/pull/4300

**Description**
Adds the ability to look at the students in progress at the adventures and the quiz. Decided not to add a hover functionality, but instead make the in-progress bubbles buttons, so teachers can click on the number to see the students currently in progress. This avoids accidentally showing privacy-sensitive information of the students. 

Video to demonstrate functionality https://www.loom.com/share/44d8ca33b2a74e549ef5dd72504bf3dd

**Relevant technical choices**


**Test instructions**
* Make sure all functionality as described in https://github.com/hedyorg/hedy/pull/4300 still works as expected.
* Click on the number bubbles behind the adventures and the quizzes, and make sure the correct students are displayed. Clicking the bubble again should remove the students. Reloading the page should also remove the students.
